syntax = "proto3";

package pegscanner.single_stock_page;

import "stock.proto";

message NewsItem {
  string title = 1;
  string url = 2;
  string source = 3;
  int64 published_at = 4;
}

message SingleStockPageRequest {
  string symbol = 1;
}

message SingleStockPageResponse {
  pegscanner.stock.Stock stock = 1;
  repeated pegscanner.stock.KLineData daily_kline = 2;
  repeated NewsItem news = 3;
}
